src/components/Nurseries/index.tsx

```diff
@@ -23,6 +23,7 @@ type NurseriesListProps = {
 const columns: TableColumnType[] = [
   { key: 'name', name: 'Name', type: 'string' },
   { key: 'description', name: 'Description', type: 'string' },
+  { key: 'timeZone', name: strings.TIME_ZONE, type: 'string' },
```

Comment:
only if opt-in enabled, we can do this dynamically in the table props maybe for now